efficientdet/det_model_fn.py

```diff
@@ -424,10 +425,13 @@ def model_fn(inputs):
 
     def metric_fn(**kwargs):
       """Returns a dictionary that has the evaluation metrics."""
+      global params
+      params = copy.deepcopy(params)
+      params['nms_configs']['score_thresh'] = 1e-3
```

Comment:
maybe just set score_thresh = 0 as default in hparams_config.py? (then you don't need to manually set score_thresh here).

The current default value 0.4 is for certain inference cases, but in reality, people should set different score_thresh for their dataset. 

WDYT?

Reply:
I agree.